tests/pubsub/testgossipsub.nim

```diff
@@ -382,7 +387,7 @@ suite "GossipSub":
     await allFuturesThrowing(nodes[0].switch.stop(), nodes[1].switch.stop())
 
     await allFuturesThrowing(nodesFut.concat())
-    check observed == 2
+    check observed == 3
```

Comment:
It's probably better to create a new test for this.

Reply:
This test seems to be related to sending messages over fanout, however, it's not clear to me what it's testing. A proper test for this validator would check it is being called how it's expected and not being called otherwise.

Reply:
Yes, there are improvements to be made. Please add a new one if that's ok.

Reply:
Done :)

Reply:
Thanks a lot.